mantidproject/mantid
# Add changing the colorbar label to figure options images tab

**Description of work.**
This PR changes the Label input in the images tab of the figure options so that instead of setting the label of the image (not very useful), it sets the label of the colorbar.

**To test:**
Make an image plot with a colorbar, either a Colorfill or Surface plot.
Open figure options, go to Images tab.
Check that the Label field shows the current colorbar label, and changing it should update the label.
For image plots without a colorbar the label input should be disabled.

Fixes #28840 

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
